renovate.json

```diff
@@ -0,0 +1,11 @@
+{
+  "extends": [
+    "config:base"
```

Comment:
Does this get triggered every minor version or every snapshot?

Reply:
@oryan-block - Renovate will run on each commit or also on a schedule.  It checks for updates from maven central.  The ootb package rules from Renovate are pretty decent.  

I had to tweak the package rule for `com.graphql-java` because they publish developer builds to maven central which we'd want to ignore.

Full extent of configuration options available - https://docs.renovatebot.com/configuration-options/

Reply:
We can update the `renovate.json` as needed to further fine tune things like
* packages matched
* version ranges we're after
* grouping of packages into the same PRs (many [sensible defaults](https://docs.renovatebot.com/presets-group/) already)
* amount of concurrent open PRs etc

Reply:
Sorry I meant to ask if this will open PRs for every patch version?

Reply:
By default yes, every patch version will be detected with a corresponding branch + PR.

Reply:
For example, here is the `graphql-spring-boot` detecting that v11.0.1 of `graphql-java-tools` was available - https://github.com/graphql-java-kickstart/graphql-spring-boot/pull/584

Reply:
Right. I would consider maybe only doing it for minor versions (maybe only for frequently updated deps?) depending on how often that this ends up happening. 
But this is ok for now.

Reply:
For reference, the config to do that would be:
```
  // disable patch updates for "dome-dep"
  "packageRules": [{
    "matchPackageNames": ["some-dep"],
    "matchUpdateTypes": ["patch"],
    "enabled": false
  }]
```